packages/gatsby-plugin-twitter/src/__tests__/gatsby-browser.js

```diff
@@ -0,0 +1,28 @@
+const { onRouteUpdate } = require(`../gatsby-browser`)
+
+describe(`gatsby-plugin-twitter`, () => {
+  describe(`onRouteUpdate`, () => {
+    it(`injects the twitter script if it hasn't already been injected`, () => {
```

Comment:
One note here, not really a blocker, but I probably would've written two tests here.

```
it(`adds the twitter script`, () => {})
it(`only adds the twitter script once`, () => {})
```

just to keep the test a little more encapsulated.

Reply:
Thanks for looking at this, will keep that in mind 🙂